debian/ubuntu-advantage-tools.postinst

```diff
@@ -95,45 +98,96 @@ if status:
 
 unconfigure_esm() {
     if ! check_service_is_enabled esm-infra; then
-        rm -f $APT_TRUSTED_KEY_DIR/ubuntu-esm*gpg  # Remove previous esm keys
-        rm -f $APT_TRUSTED_KEY_DIR/$ESM_INFRA_KEY_TRUSTY
-        rm -f $ESM_INFRA_APT_SOURCE_FILE_TRUSTY
-        rm -f $ESM_INFRA_OLD_APT_SOURCE_FILE_TRUSTY
-        rm -f $ESM_APT_PREF_FILE_TRUSTY $ESM_INFRA_OLD_APT_PREF_FILE_TRUSTY
-        rm -f $ESM_INFRA_APT_PREF_FILE_TRUSTY
+        rm -f "$APT_TRUSTED_KEY_DIR/ubuntu-esm*gpg"  # Remove previous esm keys
+        rm -f "$APT_TRUSTED_KEY_DIR/$ESM_INFRA_KEY_TRUSTY"
+        rm -f "$ESM_INFRA_APT_SOURCE_FILE"
+        rm -f "$ESM_INFRA_OLD_APT_SOURCE_FILE_TRUSTY"
+        rm -f "$ESM_APT_PREF_FILE_TRUSTY" "$ESM_INFRA_OLD_APT_PREF_FILE_TRUSTY"
+        rm -f "$ESM_INFRA_APT_PREF_FILE"
+    fi
+    if ! check_service_is_enabled esm-apps; then
+        rm -f "$APT_TRUSTED_KEY_DIR/$ESM_APPS_KEY_TRUSTY"
```

Comment:
Maybe I missing something, but it seems that `ESM_APPS_KEY_TRUSTY` doesn't exist

Reply:
+1 I've fixed this.